connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignorTest.java

```diff
@@ -1133,20 +1219,9 @@ public void testTaskAssignmentWhenTasksDuplicatedInWorkerAssignment() {
         assertDelay(0, returnedAssignments);
         expectedMemberConfigs = memberConfigs(leader, offset, returnedAssignments);
         assertNoReassignments(memberConfigs, expectedMemberConfigs);
-        assertAssignment(1, 4, 0, 2, "worker1", "worker2");
```

Comment:
This revocation is unnecessary because we revoked the connecor1 and all of the tasks in connector1 in previous round, so when entering this round, the assignment is:
W1: connectors: [connector2], tasks: [T2-0, T2-1, T2-2, T2-3]
W2: connectors: [], tasks: []

We can just assign the connector1 and 4 of his tasks into W2, and complete the rebalance.

However, before my change, in `performTasksRevocation` method, we use `activeAssignment` as the total tasks, so, we'll get the numTasksToRevoked = 4 (active tasks) / 2 (total worker) = 2, then revoke 2 tasks, and then assign to 2 workers in next round.